src/modules/libraryModule.js

```diff
@@ -14,80 +14,127 @@ export default class Library {
+    <div class="box unchecked"></div>
+    <input class="task" type="text" value="${title}" name="Item1">
+    <div class="handle removeButton" id="${id}" type="button"></div>
+        `;
+  }
+
+  static displayCheckedBook(title, id) {
+    return `
+      <div class="box checked"></div>
+      <input class="task" type="text" value="${title}" name="Item1">
+      <div class="handle removeButton" id="${id}" type="button"></div>
         `;
   }
 
   pullFromStorage = () => {
-    const parsed = JSON.parse(sessionStorage.getItem('strShelf'));
+    const parsed = JSON.parse(localStorage.getItem('strShelf'));
     let counter = 1;
     const preShelf = [];
-    for (let i = 0; i < this.shelf.length; i += 1) {
+    // Returns the length of the localstorage shelf
+    // (even when the page reloads)
+    const shelfSize = Object.values(parsed).length;
+
+    for (let i = 0; i < shelfSize; i += 1) {
+      // Returns the Nth item saved on the localstorage
+      if (this.shelf.length === 0) {
+        for (let i = 0; i < shelfSize; i += 1) {
+          this.shelf.push(Object.values(parsed)[i]);
+        }
+      }
+      // Get the completed value of parsed book
       const parsedBook = parsed[`${counter}`];
-      const tempShelf = Library.displayBook(parsedBook.title, counter);
-      preShelf.unshift(tempShelf);
+      let tempShelf = Library.displayBook(parsedBook.title, counter);
+      if (parsedBook.completed) {
+        tempShelf = Library.displayCheckedBook(parsedBook.title, counter);
+      }
+      preShelf.push(tempShelf);
       counter += 1;
     }
+    return preShelf;
+  }
 
+  paintToPage = () => {
+    const preShelf = this.pullFromStorage();
     this.frontShelf.innerHTML = '';
-
     for (let i = 0; i < preShelf.length; i += 1) {
       const createdBook = document.createElement('div');
       createdBook.classList.add('unchecked-slot', 'book');
       createdBook.innerHTML = preShelf[i];
       createdBook.id = i + 1;
       this.frontShelf.appendChild(createdBook);
     }
+    this.setRemoveListeners();
   }
 
   updateShelf = () => {
+    // console.log('bookshelf A: ' + this.bookshelf);
     this.bookshelf = new StrShelf();
+
     let counter = 0;
     for (let i = 0; i < this.shelf.length; i += 1) {
       counter += 1;
       this.shelf[i].id = counter;
+      // this.shelf[i].index = counter;
     }
     for (let i = 0; i < this.shelf.length; i += 1) {
       // GIVES THE BOOK OBJ AN ID #NUMBER
       this.bookshelf[`${this.shelf[i].id}`] = this.shelf[i];
     }
+    // console.log('bookshelf C: ' + this.bookshelf);
     Library.pushToStorage(this.bookshelf);
   }
 
   newBook = (title) => {
     this.bookshelf = new StrShelf();
     const book = new Book();
     book.title = title;
-    this.shelf.unshift(book);
+    book.completed = false;
+    this.shelf.push(book);
     this.updateShelf();
+    this.paintToPage();
+    this.setInputFieldListeners();
+    this.setCheckboxFieldListeners();
   }
```

Comment:
- [ ] It would be better if we are able to perform the following functionalities in the browser:
- [ ] Add todos
- [ ] Delete todos
- [ ] Edit todos
![Screenshot from 2022-03-27 02-56-03](https://user-images.githubusercontent.com/46388113/160264814-d12e81fe-2698-42ca-a858-7569fddb06bc.png)

Reply:
I commented a piece of the code, so it stopped working, thank you ;-)